src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -103,6 +103,55 @@ public void GetApiContractReferencesFindsVersionedWinMDs()
         }
 
         [Fact]
+        public void GetWinBlueSDKLocation()
+        {
+            string sdkRootPath = ToolLocationHelper.GetPlatformSDKLocation("Windows", "8.1");
+
+            string returnValue = ToolLocationHelper.GetSDKContentFolderPath("Windows", "8.1", null, null, null, null);
+            Assert.Equal(returnValue, sdkRootPath);
```

Comment:
You have all of your Asserts backwards, it's supposed to be expected, then actual.
``` C#
void Equal(string expected, string actual)
```